tests/runtime/basic

```diff
@@ -295,3 +295,8 @@ NAME map strings are memset
 PROG BEGIN { @x = "xxxxx"; @x = "a"; @[@x] = 1; @y = "yyyyy"; @y = "a"; @[@y] = 1; printf("len: %d\n", len(@)); exit(); }
 EXPECT len: 1
 TIMEOUT 1
+
+NAME print_per_cpu_map_vals
+PROG software:cpu-clock:1e7 { @a = avg(5); @c = count(); @s = sum(5); @mn = min(1); @mx = max(10); if (@c == 10) { print((@a, @c, @s, @mn, @mx)); exit(); } }
+EXPECT (5, 10, 50, 0, 10)
```

Comment:
Seeing this failure in CI: 
```
 [  FAILED  ] basic.print_per_cpu_map_vals
	Command: /home/runner/work/bpftrace/bpftrace/build-ci/src/bpftrace  -e 'software:cpu-clock:1e7 { @a = avg(5); @c = count(); @s = sum(5); @mn = min(1); @mx = max(10); if (@c == 10) { print((@a, @c, @s, @mn, @mx)); exit(); } }'
	Expected: (5, 10, 50, 0, 10)
	Found:
Attaching 1 probe...\n__BPFTRACE_NOTIFY_PROBES_ATTACHED\n(5, 10, 45, 0, 10)\n(5, 10, 45, 0, 10)\n\n\n@a: 5\n\n@c: 12\n@mn: 1\n@mx: 10\n@s: 55\n
```

Why is `@mn` expected to be 0? I only see assignments of 1 to it.

Also, how does `@c` come out how to 10 but `@s` to be 45? Is there some kind of race?

Reply:
Yeah that doesn't make sense that `@mn` is 0. I also don't know how sum comes out to be 45 :( - Thanks for flagging.

Reply:
Aside from the `min` issue. I think this is just a racy test. I'll investigate and fix both.

Reply:
Temp fix: https://github.com/bpftrace/bpftrace/pull/3287
Issue filed: https://github.com/bpftrace/bpftrace/issues/3286